src/Ilios/MeSH/Parser.php

```diff
@@ -10,136 +16,119 @@
- * @link http://www.nlm.nih.gov/databases/dtd/nlmdescriptorrecordset_20160101.dtd
+ * @link https://nlmpubs.nlm.nih.gov/projects/mesh/MESH_FILES/xmlmesh/DTD/2024/nlmdescriptorrecordset_20240101.dtd
  */
 class Parser
 {
     // Elements
-    const ABBREVIATION = 'Abbreviation';
-    const ALLOWABLE_QUALIFIER = 'AllowableQualifier';
-    const ALLOWABLE_QUALIFIER_LIST = 'AllowableQualifiersList';
-    const ANNOTATION = 'Annotation';
-    const CASN1_NAME = 'CASN1Name';
-    const CONCEPT = 'Concept';
-    const CONCEPT1_UI = 'Concept1UI';
-    const CONCEPT2_UI = 'Concept2UI';
-    const CONCEPT_LIST = 'ConceptList';
-    const CONCEPT_NAME = 'ConceptName';
-    const CONCEPT_RELATION = 'ConceptRelation';
-    const CONCEPT_RELATION_LIST = 'ConceptRelationList';
-    const CONCEPT_UI = 'ConceptUI';
-    const CONSIDER_ALSO = 'ConsiderAlso';
-    const DATE_CREATED = 'DateCreated';
-    const DATE_ESTABLISHED = 'DateEstablished';
-    const DATE_REVISED = 'DateRevised';
-    const DAY = 'Day';
-    const DESCRIPTOR_NAME = 'DescriptorName';
-    const DESCRIPTOR_RECORD = 'DescriptorRecord';
-    const DESCRIPTOR_RECORD_SET = 'DescriptorRecordSet';
-    const DESCRIPTOR_REFERRED_TO = 'DescriptorReferredTo';
-    const DESCRIPTOR_UI = 'DescriptorUI';
-    const ECIN = 'ECIN';
-    const ECOUT = 'ECOUT';
-    const ENTRY_COMBINATION = 'EntryCombination';
-    const ENTRY_COMBINATION_LIST = 'EntryCombinationList';
-    const ENTRY_VERSION = 'EntryVersion';
-    const HISTORY_NOTE = 'HistoryNote';
-    const MONTH = 'Month';
-    const NLM_CLASSIFICATION_NUMBER = 'NLMClassificationNumber';
-    const ONLINE_NOTE = 'OnlineNote';
-    const PHARMACOLOGICAL_ACTION = 'PharmacologicalAction';
-    const PHARMACOLOGICAL_ACTION_LIST = 'PharmacologicalActionList';
-    const PREVIOUS_INDEXING = 'PreviousIndexing';
-    const PREVIOUS_INDEXING_LIST = 'PreviousIndexingList';
-    const PUBLIC_MESH_NOTE = 'PublicMeSHNote';
-    const QUALIFIER_NAME = 'QualifierName';
-    const QUALIFIER_REFERRED_TO = 'QualifierReferredTo';
-    const QUALIFIER_UI = 'QualifierUI';
-    const REGISTRY_NUMBER = 'RegistryNumber';
-    const RELATED_REGISTRY_NUMBER = 'RelatedRegistryNumber';
-    const RELATED_REGISTRY_NUMBER_LIST = 'RelatedRegistryNumberList';
-    const SCOPE_NOTE = 'ScopeNote';
-    const SEE_RELATED_DESCRIPTOR = 'SeeRelatedDescriptor';
-    const SEE_RELATED_LIST = 'SeeRelatedList';
-    const SORT_VERSION = 'SortVersion';
-    const STRING = 'String';
-    const TERM = 'Term';
-    const TERM_LIST = 'TermList';
-    const TERM_NOTE = 'TermNote';
-    const TERM_UI = 'TermUI';
-    const THESAURUS_ID = 'ThesaurusID';
-    const THESAURUS_ID_LIST = 'ThesaurusIDlist';
-    const TRANSLATORS_ENGLISH_SCOPE_NOTE = 'TranslatorsEnglishScopeNote';
-    const TRANSLATORS_SCOPE_NOTE = 'TranslatorsScopeNote';
-    const TREE_NUMBER = 'TreeNumber';
-    const TREE_NUMBER_LIST = 'TreeNumberList';
-    const YEAR = 'Year';
+    public const ABBREVIATION = 'Abbreviation';
```

Comment:
I don't know that it matters, but are these `public`? I searched for a few of them and they seem to only be used in this class.

Reply:
PHPCS won't have it any other way. without the `public` accessor, it will emit this warning:

`Visibility must be declared on all constants if your project supports PHP 7.1 or later`

Reply:
I was assuming we'd go `protected` instead, but let's make that a next time problem.

Reply:
this is a lateral shift, as these are public by default/without the accessor. don't see why not, the values can't be tampered with.